IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate/Browser.cs

```diff
@@ -130,13 +130,37 @@ public bool WaitUntil(Func<bool> func, int secondsToWait = 15)
         /// </summary>
         public void TakeScreenshot()
         {
-            string fullPath = Path.Combine(Directory.GetCurrentDirectory(), "screenshot", $"{((RemoteWebDriver)this.Driver).Capabilities.BrowserName}_{DateTime.Now:yyyy.MM.dd_hh.mm.ss}.png");
-            Console.WriteLine($"Saving screenshot to location: {fullPath}");
-            Directory.CreateDirectory(Path.Combine(Directory.GetCurrentDirectory(), "screenshot"));
+            string path = Path.Combine(Directory.GetCurrentDirectory(), "screenshot");
```

Comment:
Yeah, I just need some common spot to dump screenshots if the user doesn't specify. I was thinking C:\Temp as another possibility. I suppose that may work better since it won't ever change.

Reply:
How important is it to be easy to find? In general you can't assume write access outside of the user profile. So typically something like local app data, or temp directory are good choices.

Reply:
Changed to C:\Temp. If I need to move it down to a user temp directory, I can do that.